Formula/atop.rb

```diff
@@ -0,0 +1,28 @@
+class Atop < Formula
+  desc "Advanced system and process monitor for Linux using process events"
+  homepage "https://www.atoptool.nl"
+  url "https://github.com/Atoptool/atop/archive/refs/tags/v2.6.0.tar.gz"
+  sha256 "134de16cd58fb651d952e8ac401b20fe7f724afc0b152f06486e5c6f7be9c31a"
+  license "GPL-2.0-or-later"
+  head "https://github.com/Atoptool/atop.git", branch: "master"
+
+  depends_on :linux
+  depends_on "ncurses"
+  depends_on "zlib"
+
+  def install
+    if build.head?
+      git_head = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
+      inreplace "version.h" do |s|
+        s.sub!(/"$/, "-#{git_head}\"")
+      end
+    end
+    system "make"
+    bin.install "atop"
+    man1.install gzip("man/atop.1")
```

Comment:
Any reason why it should install these things specifically instead of using `make install`?

Reply:
The reason is that the hand-written Makefile is hard-wired to install the files into the system itself (/usr/...).
Therefore the install is better done without the Makefile.

Reply:
Could we make a pull request upstream to allow setting `PREFIX`?